### PR TITLE
Develop

### PR DESCRIPTION
**Pull request by SashaPylypyuk**
https://sashapylypyuk.github.io/react_movies-list/

### CODE REVIEW

File: src/App.js [hunk 1]
```diff
@@ -1,177 +1,15 @@
...
-
-            <div className="content">
-              Two women troubled with guy-problems swap homes in each other's
-              countries, where they each meet a local guy and fall in love.
-              <br />
-              <a href="https://www.imdb.com/title/tt0457939">IMDB</a>
-            </div>
-          </div>
-        </div>
+        <MoviesList filmes={moviesFromServer} />
```
Comment (irenhh) [inline src/App.js]:
```suggestion
        <MoviesList films={moviesFromServer} />
```

File: src/components/MoviesList/MoviesList.js [hunk 2]
```diff
@@ -1,6 +1,25 @@
 import React from 'react';
 import './MoviesList.scss';
+import PropTypes from 'prop-types';
+import { MovieCard } from '../MovieCard/MovieCard';
 
-export const MoviesList = () => (
-  <>Put the list here</>
+export const MoviesList = ({ filmes }) => (
+  <>
```
Comment (irenhh) [inline src/components/MoviesList/MoviesList.js]: do you really need it?

File: src/components/MoviesList/MoviesList.js [hunk 3]
```diff
@@ -1,6 +1,25 @@
...
+    { filmes.map(element => (
```
Comment (irenhh) [inline src/components/MoviesList/MoviesList.js]:
```suggestion
    {filmes.map(element => (
```

Comment (irenhh) [reply src/components/MoviesList/MoviesList.js]: why element?

File: src/components/MoviesList/MoviesList.js [hunk 4]
```diff
@@ -1,6 +1,25 @@
...
+      <MovieCard {...element} />
+    ))}
+
```
Comment (irenhh) [inline src/components/MoviesList/MoviesList.js]: remove this

File: src/components/MoviesList/MoviesList.js (hunk 2 above)
Comment (SashaPylypyuk) [reply src/components/MoviesList/MoviesList.js]: Yes. Without this it doesn't work

**Pull request closed without merging**